fuzzer/modules/sql_injection/strategies/lib/auth_request.py

```diff
@@ -0,0 +1,49 @@
+from oauthlib.oauth2 import BackendApplicationClient
```

Comment:
yeah... this should be DRYed up. I might do that later, but will focus on the presentation instead for now.

Reply:
DRYed it